packages/frontend/src/components/projects/sections/l3-risk-analysis-section.tsx

```diff
@@ -0,0 +1,286 @@
+import { type Sentiment } from '@l2beat/shared-pure'
+import { type ReactNode } from 'react'
+import { UnderReviewBadge } from '~/components/badge/under-review-badge'
+import { HorizontalSeparator } from '~/components/core/horizontal-separator'
+import { Markdown } from '~/components/markdown/markdown'
+import { BigIndividualRosette } from '~/components/rosette/individual/big-individual-rosette'
+import { type RosetteValueTuple } from '~/components/rosette/individual/individual-rosette-icon'
+import { BigPizzaRosette } from '~/components/rosette/pizza/big-pizza-rosette'
+import { type RosetteValue } from '~/components/rosette/types'
+import { SentimentText } from '~/components/sentiment-text'
+import { EM_DASH } from '~/consts/characters'
+import { RoundedWarningIcon } from '~/icons/rounded-warning'
+import { ShieldIcon } from '~/icons/shield'
+import { UnverifiedIcon } from '~/icons/unverified'
+import { cn } from '~/utils/cn'
+import { sentimentToBgColor } from '~/utils/sentiment'
+import { WarningBar } from '../../warning-bar'
+import { ProjectSection } from './project-section'
+import { type ProjectSectionProps } from './types'
+
+export interface L3RiskAnalysisSectionProps extends ProjectSectionProps {
+  l2: {
+    name: string
+    risks: RosetteValueTuple
+  }
+  l3: {
+    name: string
+    risks: RosetteValueTuple
+  }
+  combined?: RosetteValueTuple
+  warning: string | undefined
+  isVerified: boolean | undefined
+  redWarning: string | undefined
+}
+
+export function L3RiskAnalysisSection({
+  l2,
+  l3,
+  combined,
+  warning,
+  isVerified,
+  redWarning,
+  ...sectionProps
+}: L3RiskAnalysisSectionProps) {
+  const isUnderReview =
+    !!sectionProps.isUnderReview ||
+    l2.risks
+      .concat(l3.risks)
+      .some(({ sentiment }) => sentiment === 'UnderReview')
+
+  return (
+    <ProjectSection {...sectionProps} isUnderReview={isUnderReview}>
+      <div className="text-sm dark:text-white/80">
+        The L3 risk rosette depends on the individual L3 properties and the
+        risks of the host chain. The combined view shows the risk based on both
+        the L2 and the L3.
```

Comment:
Change text to: The L3 risks depend on the individual properties of L3 and those of the host chain combined.

Reply:
https://github.com/l2beat/l2beat/pull/5076/commits/5d28f633b273c4a9e0d64fb2e631ead3c7d81651